tests/_sandbox/pages/models.py

```diff
@@ -18,7 +19,8 @@ class HomePage(TranslatablePage):
         'wagtailimages.Image', null=True, blank=True,
         on_delete=models.SET_NULL, related_name='+')
 
-    content_panels = TranslatablePage.content_panels + [
+    content_panels = [
```

Comment:
This could have been `Page.content_panels`, which would ensure a consistent behavior when wagtail decides to add some extra fields to content panel.